src/bootstrap.sh

```diff
@@ -18,12 +18,13 @@ clone_cicd_tools_repo() {
 }
 
 _delete_cicd_tools_rootdir() {
-    echo "Removing existing CICD tools directory: '${CICD_TOOLS_ROOTDIR}'"
+    cicd_tools::debug "Removing existing CICD tools directory: '${CICD_TOOLS_ROOTDIR}'"
```

Comment:
yeah - I just tried to follow [Google's Shell style naming convention](https://google.github.io/styleguide/shellguide.html#naming-conventions), so all credit goes to them :) 

I think I made a reference to this styleguide in the README, if not, I'll add it, because I'd like to enforce this to all scripts on the repo.

Reply:
I added a small note to the README